AgContext.yaml

```diff
@@ -0,0 +1,149 @@
+
+location_continent:
+  type:
+    anyOf:
+    - type: string
+    description: |-
+      Continent where AgContext is located
+      string. Required.
+      A string representing the continent where the AgContext is located.
+
+location_country:
+  type:
+    anyOf:
+    - type: string
+    description: |-
+      Country where AgContext is located
+      string. Required.
+      A string representing the country where the AgContext is located.
+
+location_state:
+  type:
+    anyOf:
+    - type: string
+    description: |-
+      State where AgContext is located
+      string. Required.
+      A string representing the state/province where the AgContext is located.
+
+upload_time:
+  type:
+    anyOf:
+    - type: DateTime<'Y-m-d H:i:s'>
+    description: |-
+      Data and time when this AgContext was uploaded.
+      DateTime. Required.
+      System generated date time object recorded when a dataset with AgContext is uploaded.
+
+camera_type:
+  type:
+    anyOf:
+    - type: string
+    - type: string
+      enum:
+      - variable
+    description:
+      Type of camera
+      string.
+      A string describing the model/make of camera used for this AgContext. If this is not fixed within the AgContext, use value "variable".
+
+camera_height:
+  type:
+    anyOf:
+    - type: integer
+    - type: string
+      enum:
+      - variable
+    description: |-
+      Height of camera
+      int or "variable". Required.
+      An integer representing the height of the camera, measured in millimeters . If this is not fixed within the AgContext, use value "variable".
+
+camera_angle:
+  type:
+    anyOf:
+    - type: integer
+    - type: string
+      enum:
+      - variable
+    description:
+      Angle of camera
+      int or "variable". Required
+      An integer representing the angle at which the camera is positioned, in degrees. A camera facing straight down would be 90. If this is not fixed within the AgContext, use value "variable".
+
+camera_fov:
+  type:
+    anyOf:
+    - type: integer
+      minimum: 1
+      maximum: 180
+    - type: string
+      enum:
+      - variable
+    description: |-
+      Field of view for camera
+      int or "variable". Required.
+      An integer representing the angle captured by the camera across the diagonal of an image, measured in degrees. If this is not fixed within the AgContext, use value "variable".
+
+emr_channels:
+  type:
+    anyOf:
+    - type: string
+      enum:
+      - visual
+      - NIR
+      - Red
+    description: |-
+      EMR channel recorded by camera
+      "visual", "NIR", and/or "Red".
+      A string describing the electromagnetic radiation channels recorded by the camera.
+
+url:
+  type:
+    anyOf:
+    - type: string
+    description: |-
+      URL of AgContext
+      string.
+      A string recording the URL of the AgContext.
```

Comment:
I think this kind of stuff belongs in Collection, I think

Reply:
This section would likely benefit from controlled vocabulary. for a moment I considered having wavelengths